### PR TITLE
F7, F8 & F9 Criterias merged and reference link added

### PR DESCRIPTION
**Pull request by janhalen**
@zorp please have a look at these changes and merge if the are according to the agreements on last meeting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,7 @@ Hvis feltet efterlades tomt, betragtes evalueringen som ikke færdiggjort.
...
-|  | F7  | Driftskrav til løsningen er dokumenteret                                                                                                       | Hvor?                                                               | 1         |
-|  | F8  | Løsningen er dokumenteret på teknisk niveau                                                                                                   | Hvor?                                                               | 1         |
-|  | F9  | Teknisk implementering af løsningen er dokumenteret                                                                                            | Hvor?                                                               | 1         |
+|  | F7  | Alt dokumentation til projektet udarbejdes med og overholder OS2s standardskabelon for dokumentation.                                                                                                       | [Documentation template for OS2 projects](http://github.com/OS2offdig/os2-docs-template)
```
Comment (zorp) [inline README.md:40]: Does this mean that we need a replacement demand for level 1 products as the demand for operating requirements being documented is deleted?

**Review by zorp — APPROVED** [review]
I only had one comment. The change looks good but we have to pay attention to the follow-up requirements.

File: README.md (hunk 1 above)
Comment (janhalen) [reply README.md:40]: We could add a section to the documentation template that requires a description of how you install and operate the product

Comment (janhalen) [reply README.md:40]: Work has started here: https://github.com/OS2offdig/os2-docs-template/issues/12

**Pull request merged**